DeratControl.Domain/Root/Exceptions/PerimeterMissingException.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DeratControl.Domain.Root.Exceptions
+{
+    class PerimeterMissingException : DomainException
```

Comment:
Please stay consistent with other exception names, call it smth like PerimeterNotExistException

Reply:
also make it public